framework/auth/views.py

```diff
@@ -245,13 +245,8 @@ def auth_login(auth, **kwargs):
         next_url = request.args.get('redirect_url')
         must_login_warning = False
 
-    if next_url:
```

Comment:
`validate_next_url(next_url)` will check `None` and return `True` if there is no `next_url`.
This is because in the other PR, `next_url` always have a default value if `None`